### PR TITLE
Topic recognition - s45339747

### PR DESCRIPTION
**Pull request by Striiipes**
Student: Jonathan Godbold, s45339747
Problem: Classify laterality (left or right sided knee) of the OAI AKOA knee data set having a minimum accuracy of 0.9 on the test set.
There are four files contained in the module.
Script execute_script_45339747.py is used to call the other three scripts which will format the data from the OAI AKOA dataset, build the model, train the model and plot the results (respectively).
Please see the README.md file for more information.
Please provide any feedback you have and I'll respond as soon as I can.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently descriptive commit messages
- Code and results appear valid for Easy Difficulty

Other Feedback
- Good code structure \ grouping of recourses
- Remove unused imports (e.g. `from tensorflow.keras.layers import Input, Conv2D, Dense, Activation, Flatten, Dropout, MaxPooling2D, BatchNormalization
from tensorflow.keras.models import Sequential, Model, load_model`)
- Good amount of comments

**Comment by rexsw** [comment]
Code looks good for easy difficulty

Long doc string comments such as
    Paramters:
    model - model to add layer to.
    input_shape - shape of input image.
    weight_decay - learning rate.
Ect. Are useful if you need to generate documentation for say an open-source project but also make the code harder to read so be careful with them.

Overall however your comments and code is really well laid out, good job.

The only thing I'll say is be careful with putting print statements everywhere. If I run this on a server or something print statements might be very annoying, so if you’re going to have a bunch all over the place make sure there’s some way to turn them off.

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme Good
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. remove unnecessary imports.**
In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Comment by Striiipes** [comment]
I have added your feedback. Thanks.
